src/sqlJob.ts

```diff
@@ -286,9 +281,7 @@ export class SQLJob {
       run: type === ExplainType.Run,
     };
 
-    const result = await this.send(JSON.stringify(explainRequest));
-
-    const explainResult: ExplainResults<any> = JSON.parse(result);
+    const explainResult = await this.send<ExplainResults<any>>(explainRequest);
```

Comment:
Instead of `ExplainResults<any>` should we use `ExplainResults<T>` to leave specifying `any` or ideally an actual type to the caller? This will require a change to the function signature. Plus, this change makes it more consistent with the function signature for `execute`.
```ts
async explain<T>(statement: string, type: ExplainType = ExplainType.Run): Promise<ExplainResults<T>>
```

Reply:
Good idea!